scripts/new-tor-issue

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""
+Creates a new issue to track Tor updates or appends a comment
+to any existing issues
+"""
+
+import json
+import random
+import subprocess
+import tempfile
+
+REPOSITORY = "freedomofpress/securedrop"
+# TODO: Add more
+SALUTATIONS = ["Aloha", "Bonjour", "Ciao", "Dear human overlords"]
+TEMPLATE = """\
+{salutation},
+
+A new Tor update is available.
+
+Details should be available on the [Tor forum](https://forum.torproject.net/c/news/tor-release-announcement/28).
+
+<details><summary>Here is the commit I just pushed to
+apt-test:</summary>
+
+```diff
+{patch}
+```
+</details>
+
+* [x] CI adds new packages to apt-test
+* [ ] Install tor, tor-geoipdb packages from apt-test on a prod
+      install and let them sit overnight
+* [ ] Verify that tor is still running after reboot, services
+      available, no errors or unexpected messages in logs
+* [ ] Submit a PR to `securedrop-debian-packages-lfs` to deploy
+      the same packages
+"""
```

Comment:
How would you feel about having the new ticket point to where it came from, so that the automation is self-documenting?

```suggestion

PS.  This pull request was created by `scripts/new-tor-issue` via CI job `reprepro-update-tor`.
"""
```

Reply:
Good idea! I'll push it in a minute, with `s/pull request/issue/`.